proto/info/v1/service.proto

```diff
@@ -0,0 +1,126 @@
+syntax = "proto3";
+package info.v1;
+
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/ava-labs/avalanchego/proto/pb/info/v1;infov1";
+
+// InfoService is the API service for unprivileged info on a node
+service InfoService {
+  rpc GetNodeVersion(GetNodeVersionRequest) returns (GetNodeVersionResponse);
+  rpc GetNodeID(GetNodeIDRequest) returns (GetNodeIDResponse);
+  rpc GetNodeIP(GetNodeIPRequest) returns (GetNodeIPResponse);
+  rpc GetNetworkID(GetNetworkIDRequest) returns (GetNetworkIDResponse);
+  rpc GetNetworkName(GetNetworkNameRequest) returns (GetNetworkNameResponse);
+  rpc GetBlockchainID(GetBlockchainIDRequest) returns (GetBlockchainIDResponse);
+  rpc Peers(PeersRequest) returns (PeersResponse);
+  rpc IsBootstrapped(IsBootstrappedRequest) returns (IsBootstrappedResponse);
+  rpc Upgrades(UpgradesRequest) returns (UpgradesResponse);
+  rpc Uptime(UptimeRequest) returns (UptimeResponse);
+  rpc GetVMs(GetVMsRequest) returns (GetVMsResponse);
+}
+
+message GetNodeVersionRequest {}
+message GetNodeVersionResponse {
+  string version = 1;
+  string database_version = 2;
+  uint32 rpc_protocol_version = 3;
+  string git_commit = 4;
+  map<string, string> vm_versions = 5;
+}
+
+message GetNodeIDRequest {}
+message GetNodeIDResponse {
+  string node_id = 1;
+  bytes node_pop = 2;
```

Comment:
Same comment w.r.t this reading weirdly - I think we want this to be a nested struct instead of raw bytes